requirements.txt

```diff
@@ -0,0 +1,11 @@
+jira==1.0.3
```

Comment:
Hi @brettleehari,

The design document specifically mentions that the requirements.txt file should reside under warriorframework/warhorn and not directly under warriorframework. Keeping it under warriorframework directory implies that all the packages in the requirements.txt file are necessary for warrior to run - when that is not the case.

Reply:
@bjrr I am afraid that the design document clearly mentions what i have implemented. I do not agree with this comment.

Reply:
@bjrr Moreover, as per python pip requirement.txt conventions. The file is generally placed in the base directory. One possible change that we could do is to place it in warrior/ directory as these dependency apply for Warrior directory. Users who do not use warhorn assembler would only use requirements.txt having it in warhorn is not useful from user perspective. 

Main use of requirement.txt is pip funtionality of pip -r requirement.txt. I have designed it for warhorn and pip install -r <filename> (pythonic) usage. Both cases would work well :)

what do you think?